Description: Update test-suite to skip 'hybrid' (v1 & v2 mounted simultaniously) cgroups for now.
Author: Dimitri John Ledkov <xnox@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1690125

--- a/src/tests/lxc-test-apparmor-mount
+++ b/src/tests/lxc-test-apparmor-mount
@@ -132,6 +132,7 @@
 	done
 else
 	for d in /sys/fs/cgroup/*; do
+		[ "$d" = "/sys/fs/cgroup/unified" ] && continue
 		[ -f $d/cgroup.clone_children ] && echo 1 > $d/cgroup.clone_children
 		[ ! -d $d/lxctest ] && mkdir $d/lxctest
 		chown -R $TUSER: $d/lxctest
--- a/src/tests/lxc-test-unpriv
+++ b/src/tests/lxc-test-unpriv
@@ -148,6 +148,7 @@
 	done
 else
 	for d in /sys/fs/cgroup/*; do
+		[ "$d" = "/sys/fs/cgroup/unified" ] && continue
 		[ -f $d/cgroup.clone_children ] && echo 1 > $d/cgroup.clone_children
 		[ ! -d $d/lxctest ] && mkdir $d/lxctest
 		chown -R $TUSER: $d/lxctest
--- a/src/tests/lxc-test-usernic.in
+++ b/src/tests/lxc-test-usernic.in
@@ -105,6 +105,7 @@
 	done
 else
 	for d in /sys/fs/cgroup/*; do
+		[ "$d" = "/sys/fs/cgroup/unified" ] && continue
 		[ -f $d/cgroup.clone_children ] && echo 1 > $d/cgroup.clone_children
 		[ ! -d $d/lxctest ] && mkdir $d/lxctest
 		chown -R usernic-user: $d/lxctest
